examples/data-sources/external.tf

```diff
@@ -1,9 +1,13 @@
 data "external" "example" {
-  program = ["python", "${path.module}/example-data-source.py"]
-
+  # don't forget to assure the machine running terraform commands 
+  # has the specified binary to run the program, in this example bash,
+  # could be python, powershell and etc.
```

Comment:
The `external` data source will automatically raise an error if the associated program is not found. If there are any adjustments to the `program` attribute documentation, they should be done in the attribute description. 👍 

```suggestion
```

Reply:
Thanks for the walk-through on the proccess! Just fixed it 👍🏻